types/qunit/qunit-tests.ts

```diff
@@ -214,13 +214,46 @@ QUnit.log(function( details ) {
   console.log( output );
 });
 
+QUnit.log(function( details ) {
```

Comment:
It seems like we can just do the following:

```
QUnit.log(function (details: QUnit.LogDetails) {
   // code...
});
```

In which case we can just remove these additional tests and add the types to the test which are already present for `log`, `begin`, `done`, etc...

Reply:
The argument type assertions are now combined into the function declarations